addons/hr_expense/wizard/hr_expense_split_wizard.py

```diff
@@ -0,0 +1,76 @@
+# -*- coding: utf-8 -*-
+# Part of Odoo. See LICENSE file for full copyright and licensing details.
+
+from odoo import fields, models, api, _
+from odoo.tools import float_compare
+
+class HrExpenseSplitWizard(models.TransientModel):
+    _name = 'hr.expense.split.wizard'
+    _description = 'Expense Split Wizard'
+
+    expense_id = fields.Many2one('hr.expense', string='Expense', required=True)
+    expense_split_line_ids = fields.One2many('hr.expense.split', 'wizard_id')
+    total_amount = fields.Monetary('Total Amount', compute='_compute_total_amount', currency_field='currency_id')
+    total_amount_original = fields.Monetary('Total amount original', related='expense_id.total_amount', currency_field='currency_id', help='Total amount of the original Expense that we are splitting')
+    total_amount_taxes = fields.Monetary('Taxes', currency_field='currency_id', compute='_compute_total_amount_taxes')
+    split_possible = fields.Boolean(help='The sum of after split shut remain the same', compute='_compute_split_possible')
+    currency_id = fields.Many2one('res.currency', related='expense_id.currency_id')
+
+    @api.depends('expense_split_line_ids.total_amount')
+    def _compute_total_amount(self):
+        for wizard in self:
+            wizard.total_amount = sum(wizard.expense_split_line_ids.mapped('total_amount'))
+
+    @api.depends('expense_split_line_ids.amount_tax')
+    def _compute_total_amount_taxes(self):
+        for wizard in self:
+            wizard.total_amount_taxes = sum(wizard.expense_split_line_ids.mapped('amount_tax'))
+
+
+    @api.depends('expense_split_line_ids.total_amount')
+    def _compute_split_possible(self):
+        for wizard in self:
+            wizard.split_possible = wizard.total_amount_original and (float_compare(wizard.total_amount_original, wizard.total_amount, precision_digits=2) != 0)
```

Comment:
It should also check that there are split lines, otherwise this line will crash when clicking on the button when there are no lines :innocent: 

```py
        expense_split = self.expense_split_line_ids[0]
```